### ISSUE TITLE
Cannot change order of Projects on Progress and Ranking using keyboard controls

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1216]
the existing drag and drop solution does not support keyboard users

**Comment by mosefer** [comment]
https://github.com/SortableJS/Sortable/issues/1954 has discussion of using applying keyboard manipulation to using the sortablejs library that is used by the dashboard (several of the linked items in the comments to that issue have more detailed discussions) including this article which includes a decent option https://robbymacdonell.medium.com/refactoring-a-sortable-list-for-keyboard-accessibility-2176b34a07f4

**Comment by mosefer** [comment]
fixed in #1170
